packages/flutter/test/widgets/drawer_test.dart

```diff
@@ -304,9 +304,7 @@ void main() {
     expect(buttonPressed, equals(true));
   });
 
-  testWidgets('Dismissible ModalBarrier includes button in semantic tree on iOS', (WidgetTester tester) async {
-    debugDefaultTargetPlatformOverride = TargetPlatform.iOS;
-
+  testWidgets('Dismissible ModalBarrier includes button in semantic tree', (WidgetTester tester) async {
```

Comment:
Test below this claims to explicitly test on Android in the title

Reply:
OK, I added a variant for that to make it explicit.  The only reason it is happening "on Android" is that the default target platform in testing is android.